sites/docs/src/content/docs/guidelines/graphic_design/colours.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Colour Guidelines
+subtitle: Description which colours nf-core uses
+---
+
+## nf-core green
+
+- RGB: <span class="badge" style="background-color: rgb(36,176,100);">36,176,100</span>
+- Hex: <span class="badge" style="background-color: rgb(36,176,100);">#24B064</span>
+
+## Other nf-core logo colours
+
+- Yellow:
+  - RGB: <span class="badge" style="background-color: rgb(236,220,134);">236,220,134</span>
+  - HEX: <span class="badge" style="background-color: rgb(236,220,134);">#ECDC86</span>
+- Brown:
+  - RGB: <span class="badge" style="background-color: rgb(63,43,41);">63,43,41</span>
+  - HEX: <span class="badge" style="background-color: rgb(63,43,41);">#3F2B29</span>
+- Dark green:
+  - RGB: <span class="badge" style="background-color: rgb(57,110,43);">57,110,43</span>
+  - HEX: <span class="badge" style="background-color: rgb(57,110,43);">#396E35</span>
+
+## Other non-logo colours
+
+Outside of the core colours, we recommend the use of the [Bootstrap colour palettes](https://getbootstrap.com/docs/5.0/customize/color/).
+
+For example, the dark backgrounds of the nf-core website uses the `$grey-50` colour palette:
+
+- gray-dark: `#343a40`
+- gray-100: `#f8f9fa`
+- gray-200: `#e9ecef`
+- gray-300: `#dee2e6`
+- gray-400: `#ced4da`
+- gray-500: `#adb5bd`
+- gray-600: `#6c757d`
+- gray-700: `#495057`
+- gray-800: `#343a40`
+- gray-900: `#212529` (main nf-core website background colour)
+
+Other main colours:
+
+- Blue (primary): `#0d6efd`
+- Grey (secondary): `#6c757d`
+- Green (success): `#22ae63`
+- Light blue (info): `#17a2b8`
+- Red (warning): `#ffc107`
+- Yellow (danger): `#dc3545`
```

Comment:
```suggestion
```
The colors are different for light theme and for dark theme, so I wouldn't list them here to avoid confusion